src/components/control/date.jsx

```diff
@@ -35,9 +36,8 @@ function fromFullDateFormat(string) {
 const Date = forwardRef(function Date(props, forwardedRef) {
   const ref = useForwardedRef(forwardedRef);
   const [active, setActive] = useState(false);
-  const [editing, setEditing] = useState(!!props.validationMessage);
+  const [editing, setEditing] = useState(true);
```

Comment:
According to the new design spec, we do not show error messages on first page load. Only on blur. So maybe we don't care about the mounted state?